### ISSUE TITLE
core resize filter Incompatible with cloud filesystem disk

### ISSUE THREAD

**Issue by everyx** [octobercms/october#5405]
- OctoberCMS Build: use composer create-project with version v1.1.1
- PHP Version: 4.7
- Database Engine: MySQL 8
- Plugins Installed: 
  * Ompmega.MixHelper
  * Martin.Forms
  * Mohsin.txt
  * VojtaSvoboda.TwigExtensions
  * October.Drivers
  * Zaxbux.DOStorageDriver

### Description:
I set Filesystem Disk to digitalocean (with plugin Zaxbux.DOStorageDriver), and `uploads` and `media` get works, but `resized` still user `local` filesystem, config is as below:

filesystems.php
```yaml
    'disks' => [

        'local' => [
            'driver' => 'local',
            'root'   => storage_path('app'),
            'url'    => '/storage/app',
        ],

        's3' => [
            'driver' => 's3',
            'key'    => env('FILESYSTEM_S3_KEY', 'your-key'),
            'secret' => env('FILESYSTEM_S3_SECRET', 'your-secret'),
            'region' => env('FILESYSTEM_S3_REGION', 'your-region'),
            'bucket' => env('FILESYSTEM_S3_BUCKET', 'your-bucket'),
        ],

        'rackspace' => [
            'driver'    => 'rackspace',
            'username'  => 'your-username',
            'key'       => 'your-key',
            'container' => 'your-container',
            'endpoint'  => 'https://identity.api.rackspacecloud.com/v2.0/',
            'region'    => 'IAD',
        ],

        'digitalocean' => [
            'driver' => 'do_spaces',
            'key'    => env('FILESYSTEM_DO_KEY', 'your-key'),
            'secret' => env('FILESYSTEM_DO_SECRET', 'your-secret'),
            'region' => env('FILESYSTEM_DO_REGION', 'your-region'),
            'space'  => env('FILESYSTEM_DO_SPACE', 'your-space'),
        ],

    ],
```

cms.php
```yaml
    'storage' => [

        'uploads' => [
            'disk'            => env('FILESYSTEM_DEFAULT', 'local'),
            'folder'          => env('FILESYSTEM_ROOT', '') . 'uploads',
            'path'            => env('FILESYSTEM_ROOT_URL', '/storage/app/') . 'uploads',
            'temporaryUrlTTL' => 3600,
        ],

        'media' => [
            'disk'   => env('FILESYSTEM_DEFAULT', 'local'),
            'folder' => env('FILESYSTEM_ROOT', '') . 'media',
            'path'   => env('FILESYSTEM_ROOT_URL', '/storage/app/') . 'media',
        ],

        'resized' => [
            'disk'   => env('FILESYSTEM_DEFAULT', 'local'),
            'folder' => env('FILESYSTEM_ROOT', '') . 'resized',
            'path'   => env('FILESYSTEM_ROOT_URL', '/storage/app/') . 'resized',
        ],

    ],
```

### Steps To Reproduce:
See above

**Comment by everyx** [comment]
Related pull request: https://github.com/octobercms/october/pull/5231

**Comment by mjauvin** [comment]
what did you set `default` to in filesystems.php? should be:
```
    'default' => 'digitalocean', 
```

**Comment by everyx** [comment]
> what did you set `default` to in filesystems.php? should be:
> 
> ```
>     'default' => 'digitalocean', 
> ```

@mjauvin Yes, I have already set the `default`:

```yaml
'default' => env('FILESYSTEM_DEFAULT', 'local'),
```

**Comment by mjauvin** [comment]
@everyx Check if [this PR](https://github.com/octobercms/october/pull/5379) resolves your issues:

**Comment by everyx** [comment]
> @everyx Check if [this PR](https://github.com/octobercms/october/pull/5379) resolves your issues:

I can’t confirm because I encountered this problem https://github.com/octobercms/october/issues/5409, can I have a workaround for this?

**Comment by LukeTowers** [comment]
@everyx the quick workaround is to load the drivers plugin via composer instead of the marketplace.

**Comment by everyx** [comment]
> @everyx the quick workaround is to load the drivers plugin via composer instead of the marketplace.

@mjauvin After a simple test, https://github.com/octobercms/october/pull/5379 should solve the problem :), so which version will it be released?

**Comment by LukeTowers** [comment]
@everyx it will be available in the next update, v1.1.2.

**Comment by LukeTowers** [comment]
Fixed by #5379
